articles/integrations/aws/aws-api-setup.md

```diff
@@ -0,0 +1,126 @@
+---
+description: How to enable delegated authentication with AWS APIs.
+url: /aws-api-setup
+toc: true
+---
```

Comment:
Maybe it's me but I didn't understand the business case or how to test this configuration